src/app/src/main/kotlin/org/vaccineimpact/api/app/logic/CoverageLogic.kt

```diff
@@ -148,6 +153,18 @@ class RepositoriesCoverageLogic(private val modellingGroupRepository: ModellingG
         return SplitData(splitData.structuredMetadata, tableData)
     }
 
+    private fun mapVaccineForCoverageSet(vaccine: String) : List<String>
+    {
+        // GAVI may provide vaccine programs rather tha individual vaccines in some coverage rows
```

Comment:
Super pedantic but these aren't vaccine programs, they're just combination vaccines (programs are the dose schedules, like MR1 vs MR2)

Reply:
Oh ok - but the things we're mapping include MR1 and MR2, so do you mean those are vaccine programs but Pentavalent is just a 3-in-1 vaccine combo, so calling them vaccine combinations would be more general and cover both?

Reply:
`MR` is also a combo vaccine - measles and rubella in one, but it's delivered in 2 doses, hence 1 and 2.